CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.29.0
 
 * Dependency updates (Vert.x 4.5.7, Netty 4.1.108.Final)
+* Fixed No schema matches error when using the `json` embedded format and the record key or value is an array.
```

Comment:
I would reword and changing in terms of added support for key and value to be JSON array and not fixed bug imho.

Reply:
Maybe something like `Added support for key and value to be JSON array. Works with `json` embedded format.` would better?

Reply:
Maybe a mix of that like "Added support for records key and value to be JSON array when using the `json` embedded format".

Reply:
Nice one!